cttportfolio.html

```diff
@@ -0,0 +1,173 @@
+<!doctype html>
+<html lang= "en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="UTF-8">
+	<title>Christian Turner: Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="cttportfolio.css">
+	<link rel="stylesheet" type="text/css" href="normalize.css">
```

Comment:
normalize needs to go above your other style sheets, or it can override your styles